packages/terra-spacer/docs/README.md

```diff
@@ -0,0 +1,71 @@
+# Terra Spacer
+
+This component is used to provide margin and/or padding space between two components based on the given values.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-spacer`
+  - `yarn add install terra-spacer`
+
+## Naming Convention
+
+| String Name   | Computed PX Value |
+| ------------- | -------------     |
+| none          | 0px               |
+| small-2       | 3px               |
+| small-1       | 5px               |
+| small         | 7px               |
+| medium        | 10px              |
+| large         | 12px              |
+| large+1       | 15px              |
+| large+2       | 20px              |
+| large+3       | 30px              |
+| large+4       | 50px              |
+
+## Usage
+
+```jsx
+import React from 'react';
+import Spacer from 'terra-spacer';
+
+// Small view ports
```

Comment:
These comments aren't accurate. Update to "Small spacing", "Medium Spacing", etc.

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/b2ec65a5ad9f9f5b81c9394939eb477f56b21774#diff-4a98913914f72896bd7b2c1918412f03R31